docs/bigquery-usage.rst

```diff
@@ -512,7 +506,7 @@ First, create the job locally:
    >>> client = bigquery.Client()
    >>> source_table = dataset.table(name='person_ages')
    >>> destination_table = dataset.table(name='person_ages_copy')
-   >>> job = source_table.copy_to(destination_table)  # API request
+   >>> job = client.copy_table('copy-table', destination_table, source_table)
```

Comment:
>Is this no longer an API request? If it still is, why drop # API Request?

`client.copy_table` creates a `CopyTableJob` instance locally (no API request made).  The job must be kicked off via the `job.begin()` method, which triggers an API request.

> Why Client.copy_table instead of Table.copy? (Sorry if we've already discussed, it's out of my memory by now.)

Because the real, underlying thing we are doing is creating a job instance, not performing the copy.  Tables don't know about jobs (but jobs do know about tables).

#1268 tracks the issue about `job_name`.